### PR TITLE
Added Discord.JS-Light

### PR DESCRIPTION
**Pull request by FAYStarNext**

### CODE REVIEW

**Comment by Nonolanlan1007** [comment]
Hi!
Thanks for your pull request. Can you explain me what is preventing you from using `discord-analytics/discordjs`? I don't see any major changes in your edits that could prevent the package from working with `discord.js-light`.

**Comment by FAYStarNext** [comment]
HI, I use discord-analytics/discordjs and it uses discordjs version 14 so people who use version 13 or discordjs-light don't update the interactions section because in discordjs-light interaction.type is returned as a string but in discordjs version 14 it is returned as a Number.

**Comment by FAYStarNext** [comment]
Discord.js version 14 
image
Discordjs-light
image

**Comment by Nonolanlan1007** [comment]
Okay 👌

**Review by Nonolanlan1007 — CHANGES_REQUESTED** [review]

File: package.json [hunk 1]
```diff
@@ -42,6 +43,7 @@
     "typescript": "^5.5.3"
   },
   "dependencies": {
+    "discord.js-light": "^4.10.0",
```
[the remark this answers is not in the document]
Comment (FAYStarNext) [reply package.json]: Okay

**Comment by FAYStarNext** [comment]
I fixed it🤔

**Review by Nonolanlan1007 — APPROVED** [review]

**Pull request merged**